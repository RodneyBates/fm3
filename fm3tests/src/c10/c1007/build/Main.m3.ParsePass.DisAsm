 ItkEOF()
 ItkLeftEnd()
 ItkBlockLt(L1(16_1),[13,2])
 ItkBlockDeclListLt(L6(16_6),[13,2])
 ItkTypeDeclLt([13,7])
 ItkDeclNo(D1,[13,7])
 ItkTypeDeclEquals([13,9])
 ItkReservedId(Rid"INTEGER",[13,11])
 ItkTypeDeclRt([13,7])
 ItkBlockDeclListSep(L1(16_1),[14,0])
 ItkTypeDeclLt([15,7])
 ItkDeclNo(D2,[15,7])
 ItkTypeDeclEquals([15,9])
 ItkReservedId(Rid"REAL",[15,11])
 ItkTypeDeclRt([15,7])
 ItkBlockDeclListSep(L2(16_2),[16,0])
 ItkTypeDeclLt([17,7])
 ItkDeclNo(D3,[17,7])
 ItkTypeDeclEquals([17,9])
 ItkReservedId(Rid"BOOLEAN",[17,11])
 ItkTypeDeclRt([17,7])
 ItkBlockDeclListSep(L3(16_3),[19,0])
 ItkTypeDeclLt([20,4])
 ItkDeclNo(D4,[20,4])
 ItkTypeDeclEquals([20,7])
 ItkReservedId(Rid"INTEGER",[20,9])
 ItkTypeDeclRt([20,4])
 ItkBlockDeclListSep(L1(16_1),[21,2])
 ItkTypeDeclLt([22,4])
 ItkDeclNo(D5,[22,4])
 ItkTypeDeclEquals([22,7])
 ItkReservedId(Rid"REAL",[22,9])
 ItkTypeDeclRt([22,4])
 ItkBlockDeclListSep(L2(16_2),[23,2])
 ItkTypeDeclLt([24,4])
 ItkDeclNo(D6,[24,4])
 ItkTypeDeclEquals([24,7])
 ItkReservedId(Rid"BOOLEAN",[24,9])
 ItkTypeDeclRt([24,4])
 ItkBlockDeclListRt(L6(16_6),[13,2])
 ItkBlockBEGIN(L1(16_1),[26,2])
 ItkStmtListRt(L0(16_0),[27,2])
 ItkStmtListLt(L0(16_0),[27,2])
 ItkBlockRt(L1(16_1),[13,2])
 ItkRightEnd()
 ItkBOF()
     0
