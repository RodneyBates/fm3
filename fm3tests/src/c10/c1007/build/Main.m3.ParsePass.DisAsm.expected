   221 16_00000000000000a2                   162 ItkEOF()
   219 16_00000000000000a3                   163 ItkLeftEnd()
   217 16_000000000000016c                   364 ItkExportIdListLt(L1(16_1),[9,7])
   212 16_0000000000000173                   371 ItkExportIdListElem(Id1"Main",[9,7])
   207 16_000000000000016f                   367 ItkExportIdListRt(L1(16_1),[9,7])
   202 16_00000000000000ea                   234 ItkBlockLt(L1(16_1),[11,2])
   197 16_00000000000001c4                   452 ItkBlockDeclListLt(L6(16_6),[11,2])
   192 16_00000000000000f3                   243 ItkTypeDeclLt([11,7])
   188 16_00000000000000bb                   187 ItkDeclNo(D1,[11,7])
   183 16_00000000000000f7                   247 ItkTypeDeclEquals([11,9])
   179 16_00000000000000b8                   184 ItkReservedId(Rid"INTEGER",[11,11])
   173 16_00000000000000f6                   246 ItkTypeDeclRt([11,7])
   169 16_00000000000001c8                   456 ItkBlockDeclListSep(L1(16_1),[13,2])
   164 16_00000000000000f3                   243 ItkTypeDeclLt([13,7])
   160 16_00000000000000bb                   187 ItkDeclNo(D2,[13,7])
   155 16_00000000000000f7                   247 ItkTypeDeclEquals([13,9])
   151 16_00000000000000b8                   184 ItkReservedId(Rid"REAL",[13,11])
   145 16_00000000000000f6                   246 ItkTypeDeclRt([13,7])
   141 16_00000000000001c8                   456 ItkBlockDeclListSep(L2(16_2),[15,2])
   136 16_00000000000000f3                   243 ItkTypeDeclLt([15,7])
   132 16_00000000000000bb                   187 ItkDeclNo(D3,[15,7])
   127 16_00000000000000f7                   247 ItkTypeDeclEquals([15,9])
   123 16_00000000000000b8                   184 ItkReservedId(Rid"BOOLEAN",[15,11])
   117 16_00000000000000f6                   246 ItkTypeDeclRt([15,7])
   113 16_00000000000001c8                   456 ItkBlockDeclListSep(L3(16_3),[17,2])
   108 16_00000000000000f3                   243 ItkTypeDeclLt([18,4])
   104 16_00000000000000bb                   187 ItkDeclNo(D4,[18,4])
    99 16_00000000000000f7                   247 ItkTypeDeclEquals([18,7])
    95 16_00000000000000b8                   184 ItkReservedId(Rid"INTEGER",[18,9])
    89 16_00000000000000f6                   246 ItkTypeDeclRt([18,4])
    85 16_00000000000001c8                   456 ItkBlockDeclListSep(L1(16_1),[20,4])
    80 16_00000000000000f3                   243 ItkTypeDeclLt([20,4])
    76 16_00000000000000bb                   187 ItkDeclNo(D5,[20,4])
    71 16_00000000000000f7                   247 ItkTypeDeclEquals([20,7])
    67 16_00000000000000b8                   184 ItkReservedId(Rid"REAL",[20,9])
    61 16_00000000000000f6                   246 ItkTypeDeclRt([20,4])
    57 16_00000000000001c8                   456 ItkBlockDeclListSep(L2(16_2),[22,4])
    52 16_00000000000000f3                   243 ItkTypeDeclLt([22,4])
    48 16_00000000000000bb                   187 ItkDeclNo(D6,[22,4])
    43 16_00000000000000f7                   247 ItkTypeDeclEquals([22,7])
    39 16_00000000000000b8                   184 ItkReservedId(Rid"BOOLEAN",[22,9])
    33 16_00000000000000f6                   246 ItkTypeDeclRt([22,4])
    29 16_00000000000001c7                   455 ItkBlockDeclListRt(L6(16_6),[11,2])
    24 16_00000000000000ee                   238 ItkBlockBEGIN(L1(16_1),[24,2])
    19 16_00000000000001cf                   463 ItkStmtListRt(L0(16_0),[25,2])
    14 16_00000000000001cc                   460 ItkStmtListLt(L0(16_0),[25,2])
     9 16_00000000000000ed                   237 ItkBlockRt(L1(16_1),[11,2])
     4 16_00000000000000a4                   164 ItkRightEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
